assets/icons/dark/videocam.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24">

```

Comment:
could these match the other icons? So newline at the end, `fill` before `d`, and use rgb instead of color name?

Reply:
Ok, I'll change it. I'm working on a separate proposal to do icon coloring as part of the theme instead of hard-coding them.